### DISCUSSION TITLE
White screen / frozen screen when sending a lot of requests

### DISCUSSION THREAD

**Discussion by VladimirLuzhin** [cefsharp/CefSharp#4671] [General]
### Is there an existing issue for this?

- [X] I have searched both open/closed issues, no issue already exists.

### CefSharp Version

120.1.10

### Operating System

Windows 10

### Architecture

x64

### .Net Version

.NET6

### Implementation

WinForms

### Reproduction Steps

I have a react application running in a CEF browser, which periodically sends various requests (healthchecks, updating settings, etc.), everything works fine, but after some time, the CEF browser dies, in devtools it looks like a "Render error process gone", and the browser either freezes or just shows a white screen.

I have prepared code that allows you to reproduce this problem. On my machine the ceph browser dies after ~800 seconds.

```
export const App: React.FC = () => {
    const [seconds, setSeconds] = useState(0)

    useEffect(() => {
        let isHandling = false;

        const interval = setInterval(async () => {
            if (isHandling)
                return

            isHandling = true
            await fetch('any available endpoint');
            isHandling = false
        }, 1)
        
        const interval2 = setInterval(() => {
            setSeconds(prev => prev + 1)
        }, 1_000)

        return () => {
            clearInterval(interval)
            clearInterval(interval2)
        }
    }, []);
    
    return <div style={{margin: 'auto'}}>{seconds} seconds have passed</div>
}
```
Its reproducing with CefSharp.MinimalExample.WinForms, i just open the page contains only this react component.

### Expected behavior

Works fine no matter how much time has passed

### Actual behavior

White / frozen screen after some time

### Regression?

Same behavior on 113.1.40 version

### Does this problem also occur in the CEF Sample Application

Yes using WinForms command line args

### Other information

last lines in debug.log
```
[0111/174545.505:ERROR:CONSOLE(0)] "ReferenceError: dragEvent is not defined", source:  (0)
[0111/174630.418:ERROR:CONSOLE(0)] "ReferenceError: dragEvent is not defined", source:  (0)
[0111/174916.662:ERROR:CONSOLE(0)] "ReferenceError: dragEvent is not defined", source:  (0)
[0111/175010.613:ERROR:command_buffer_proxy_impl.cc(319)] GPU state invalid after WaitForGetOffsetInRange.
[0111/175010.614:ERROR:command_buffer_proxy_impl.cc(127)] ContextResult::kTransientFailure: Failed to send GpuControl.CreateCommandBuffer.
[0111/175014.146:ERROR:CONSOLE(0)] "ReferenceError: dragEvent is not defined", source:  (0)
[0111/175028.141:ERROR:CONSOLE(0)] "ReferenceError: dragEvent is not defined", source:  (0)
[0111/175031.370:ERROR:CONSOLE(0)] "ReferenceError: dragEvent is not defined", source:  (0)
```

Thank you in advance!

**Comment by amaitland** [comment]
> ### Does this problem also occur in the CEF Sample Application
> Not Tested

You'll need to test with the `CEF Sample Application`.

> [0111/175010.613:ERROR:command_buffer_proxy_impl.cc(319)] GPU state invalid after WaitForGetOffsetInRange.
> [0111/175010.614:ERROR:command_buffer_proxy_impl.cc(127)] ContextResult::kTransientFailure: Failed to send GpuControl.CreateCommandBuffer.

If the `GPU Process` crashed then a white/frozen screen is to be expected.

**Reply by VladimirLuzhin** [reply]
My bad, i choose the wrong option in "Does this problem also occur in the CEF Sample Application" selection.

Its reproducing with CefSharp.MinimalExample.WinForms, i just open the page contains only this react component.

**Comment by amaitland** [comment]
To help determine where the problem needs to be fixed please download and test using the `CEF Sample Application(cefclient)`.
          1. Download for [x86](https://cef-builds.spotifycdn.com/cef_binary_120.1.8%2Bge6b45b0%2Bchromium-120.0.6099.109_windows32_client.tar.bz2) or [x64](https://cef-builds.spotifycdn.com/cef_binary_120.1.8%2Bge6b45b0%2Bchromium-120.0.6099.109_windows64_client.tar.bz2) or [arm64](https://cef-builds.spotifycdn.com/cef_binary_120.1.8%2Bge6b45b0%2Bchromium-120.0.6099.109_windowsarm64_client.tar.bz2).
          2. Extract tar.bz2 file
          3. Execute cefclient.exe using the **command line args below**:
          
          For WPF/OffScreen `cefclient.exe --multi-threaded-message-loop --no-sandbox --off-screen-rendering-enabled --enable-gpu`
          For WinForms `cefclient.exe --multi-threaded-message-loop --no-sandbox`

**Reply by xuwangyi** [reply]
Level: Error

Message: Permissions policy violation: unload is not allowed in this document.
![image](https://github.com/cefsharp/CefSharp/assets/73223745/3d1cd169-4d35-41ec-b7e5-93b9949fb628)


happen this error, how to fixed?
windows 10 ,and the screen white ,some time show website,but can't do any thing in website
the cefsharp.wpf version was 103.0.90.0

![image](https://github.com/cefsharp/CefSharp/assets/73223745/2701ff1c-acce-4f46-8197-f959afee0142)

**Reply by amaitland** [reply]
If the problem reproduces in the current supported version (`M120` at time of writing) then open your own discussion.

https://github.com/cefsharp/CefSharp/discussions/new/choose

You can use https://github.com/cefsharp/CefSharp.MinimalExample for testing purposes.

**Reply by VladimirLuzhin** [reply]
Same behaviour with CEF Sample Application(cefclient), frozen screen. Checked the versions x86 and x64

Last logs in debug.log

```
[0125/124124.535:ERROR:command_buffer_proxy_impl.cc(319)] GPU state invalid after WaitForGetOffsetInRange.
[0125/124124.535:ERROR:command_buffer_proxy_impl.cc(127)] ContextResult::kTransientFailure: Failed to send GpuControl.CreateCommandBuffer.
```

**Reply by amaitland** [reply]
As it reproduces with `cefclient` then it's not a bug in `CefSharp`.

What is your `GPU`? Try disabling `GPU Acceleration` and/or` GPU Componsiting`

```
settings.CefCommandLineArgs.Add("disable-gpu");
settings.CefCommandLineArgs.Add("disable-gpu-compositing");
```
